src/containers/MainScreen/index.js

```diff
@@ -85,19 +133,27 @@ class MainScreen extends Component {
                 coordinate={{ latitude, longitude }}
                 image={targetIcon}
                 title={title}
+                onPress={this.onPressTarget}
```

Comment:
You could do this, but I like how you get the id from the event, I didn't know that.
```suggestion
                onPress={() => this.onPressTarget(id)}
```

Reply:
Agree.